packages/smooth_app/lib/pages/product/product_image_viewer.dart

```diff
@@ -179,57 +179,66 @@ class _ProductImageViewerState extends State<ProductImageViewer> {
               )
             ],
           ),
-          Row(
-            mainAxisAlignment: MainAxisAlignment.spaceBetween,
-            crossAxisAlignment: CrossAxisAlignment.center,
-            mainAxisSize: MainAxisSize.max,
-            children: <Widget>[
-              Expanded(
-                child: Padding(
-                  padding: const EdgeInsets.symmetric(horizontal: SMALL_SPACE),
-                  child: ProductImageServerButton(
-                    product: _product,
-                    imageField: widget.imageField,
-                    language: widget.language,
-                    isLoggedInMandatory: true,
+          Container(
```

Comment:
As you only use a `Container`, could you switch to a `Padding` one instead, please?